Modules/Filtering/DiffusionTensorImage/test/itkDiffusionTensor3DReconstructionImageFilterTest.cxx

```diff
@@ -118,7 +118,33 @@ itkDiffusionTensor3DReconstructionImageFilterTest(int argc, char * argv[])
       gradientDirection[2] = gradientDirections[i][2];
       tensorReconstructionFilter->AddGradientImage(gradientDirection, gradientImage);
       std::cout << "Gradient directions: " << gradientDirection << std::endl;
+
+      const TensorReconstructionImageFilterType::GradientDirectionType::element_type epsilon = 1e-3;
```

Comment:
Not sure why such a low precision is required here, but it was the case when checking the expected and obtained values.